eskipfile/watch_test.go

```diff
@@ -45,18 +44,12 @@ func deleteFile(t *testing.T) {
 }
 
 func createFileWith(content string, t *testing.T) {
-	tmpfile, err := ioutil.TempFile("", "watch_test-*")
+	f, err := os.Create(testWatchFile)
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.Remove(tmpfile.Name())
-
-	_, err = tmpfile.WriteString(content)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	err = os.Rename(tmpfile.Name(), testWatchFile)
```

Comment:
This rename was introduced by #1555 due to race condition between `os.Create` and `f.WriteString`. Your change basically reverts that back :) See #1555 on how to reproduce the race.

What we need probably is to create/truncate `testWatchFile + ".tmp"` on the same partition and then rename it to `testWatchFile`